client/pages/upload.js

```diff
@@ -381,6 +423,31 @@ const Upload = ({ token, userId }) => {
             </Text>
           )}
         </Box>
+        <Box mb={4}>
```

Comment:
Please make this field optional. Currently I cannot submit the form if I do not upload an image. Please also move the field down to the bottom of the form, above tags